src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpConnection.cs

```diff
@@ -26,22 +26,20 @@ internal partial class SmtpConnection
         private bool _isConnected;
         private bool _isClosed;
         private bool _isStreamOpen;
-        private readonly EventHandler _onCloseHandler;
-        internal SmtpTransport _parent;
-        private readonly SmtpClient _client;
-        private NetworkStream _networkStream;
-        internal TcpClient _tcpClient;
-        internal string _host = null;
+        private readonly EventHandler? _onCloseHandler;
+        internal SmtpTransport? _parent;
+        private readonly SmtpClient? _client;
+        private NetworkStream? _networkStream;
+        internal TcpClient? _tcpClient;
```

Comment:
NIT: I might miss something but `_client` and `_tcpClient` doesn't seem to be nullable

Reply:
SmtpConnection is partial, and it has this ctor in another file:
https://github.com/dotnet/runtime/blob/5e6711dbfb1b696e1fb7c53f9d84d33c60dc87be/src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpConnection.Auth.cs#L35-L38